index.js

```diff
@@ -75,7 +61,16 @@ export default class ClapprStats extends ContainerPlugin {
 
   stopReporting() {
     this._buildReport()
+
     clearInterval(this._intervalId)
+    this._newMetrics()
+
+    this.container.off(Events.CONTAINER_TIMEUPDATE, this.onContainerUpdateWhilePlaying)
```

Comment:
@leandromoreira `stopListening` + `bindEvents` worked like a charm! Thanks for the tip!